Resources/Audio/Machines/attributions.yml

```diff
@@ -152,3 +152,24 @@
   license: "CC0-1.0"
   copyright: "dakamakat on freesound.org"
   source: "https://freesound.org/people/Dakamakat/sounds/717370/"
+
+- files: ["twobeep.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/twobeep.ogg"
+
+- files: ["chime.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/chime.ogg"
+
+- files: ["buzz-sigh.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/buzz-sigh.ogg"
+
+- files: ["buzz-two.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/buzz-two.ogg"
+
```

Comment:
Put it in the same attribution and just link the commit.

Reply:
It should be a yaml list not one string, either 
[] syntax or 
\- entry 1 
\- entry 2
syntax